kafka/src/main/java/io/micronaut/configuration/kafka/annotation/ErrorStrategy.java

```diff
@@ -61,13 +63,29 @@
     String retryDelay() default DEFAULT_DELAY_IN_SECONDS + "s";
 
     /**
-     * The retry count used with RETRY_ON_ERROR and RETRY_EXPONENTIALLY_ON_ERROR
+     * The fixed retry count used with RETRY_ON_ERROR and RETRY_EXPONENTIALLY_ON_ERROR
      * {@link io.micronaut.configuration.kafka.annotation.ErrorStrategyValue}.
      *
+     * <p>{@code retryCount} takes precedence over {@code retryCountValue} if they are both set.
+     *
      * @return the retry count of how many attempts should be made
+     * @see ErrorStrategy#retryCountValue()
      */
+    @AliasFor(member = "retryCountValue")
```

Comment:
This alias is not needed, assuming we would use only one field to read the value